articles/portal/ptl-gs.md

```diff
@@ -142,17 +138,17 @@ The Portal navigation panel provides access to all the functionality available i
 
 - **Reports** - Provides access to the following reports:
 
-  - **Login History** - Lists everyone who has logged in to the Portal over the last 30 days (see [*Viewing login information*](ptl-how-view-login-info.md)).
+  - **Login History** - Lists everyone who has logged in to the Portal over the last 30 days (see [*How to view login information*](ptl-how-view-login-info.md)).
 
   - **VMware Cloud Build History** - Lists recent compute service, VDC and edge gateway builds initiated from the Portal and the Portal API (see the [*Getting Started Guide for UKCloud for VMware*](../vmware/vmw-gs.md) and [*How to build an edge gateway*](../vmware/vmw-how-build-edge.md)).
 
   - **Backup Summary** - Provides access to summary reports of Snapshot Protection status for each of the account's compute services (see [*How to manage Snapshot Protection for your VMs*](../vmware/vmw-how-manage-snapshot-protection.md)).
 
 - **Settings** - (Portal administrators only) Change various Portal settings, including key contacts, password expiration period, two factor authentication and IP address restrictions (see [*UKCloud Portal security settings*](ptl-ref-security-settings.md)).
 
-- **Contacts** - Lists the user accounts with access to the current customer account. You can view a list of all user accounts (**All Contacts**) or just those with administration permissions (**Admin Contacts**). Portal administrators can create new users, reset other users' passwords and change other users' Portal permissions (see [*Portal permissions*](ptl-ref-overview-permissions.md)).
+- **Contacts** - Lists the user accounts with access to the current customer account. You can view a list of all user accounts (**All Contacts**) or just those with administration permissions (**Admin Contacts**). Portal administrators can create new users, reset other users' memorable words and change other users' Portal permissions (see [*Portal permissions*](ptl-ref-overview-permissions.md)).
 
-- **VMware Cloud** - Provides access to UKCloud for VMware compute services. Expand this option and then select the compute service with which you want to work (see the [*Getting Started Guide for UKCloud for VMware*](../vmware/vmw-gs.md)).
+- **VMware Cloud** - Provides access to UKCloud for VMware compute services. Expand this option and then select the compute service that want to work with (see the [*Getting Started Guide for UKCloud for VMware*](../vmware/vmw-gs.md)).
```

Comment:
I think this should say 'that you want to work with'

Reply:
It should indeed! Thanks for spotting. All fixed.